source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,120 @@
 Desktop Application Changelog
 ========================================
 
+Release 4.0.0
+--------------
+
+Release date: January 23, 2018
```

Comment:
Hm, let's change the date to next Monday. We may need to change it in the desktop app repo too.